OrbitGl/LiveFunctionsDataView.cpp

```diff
@@ -246,3 +280,95 @@ Function& LiveFunctionsDataView::GetFunction(unsigned int a_Row) const {
   CHECK(m_Functions[m_Indices[a_Row]]);
   return *m_Functions[m_Indices[a_Row]];
 }
+
+void LiveFunctionsDataView::GoToFirst(Function& function) {
+  TextBox* box_to_go;
```

Comment:
Can you initialize this pointer to `nullptr`? Technically not needed, but it makes static analyzers happy and might help with a future bug.

Reply:
Thanks!